client/components/PetTile.js

```diff
@@ -5,13 +5,15 @@ import { Link } from "react-router-dom";
 const PetTile = (props) => {
   const { id, name, imageUrl, age, vaccinationStatus } = props.pet;
   return (
-    <div >
-      <Link to={`/pets/${props.type}/${id}`}>{name}</Link>
+    <div className="tile">
       <Link to={`/pets/${props.type}/${id}`} >
         <img src={imageUrl} alt={props.type} />
       </Link>
+      <h4>
+        <Link to={`/pets/${props.type}/${id}`}>{name}</Link>
```

Comment:
these 2 Links are to the same route, we can condense to one Link wrapping around both pieces of info!

Reply:
Wow! I didn't know you could do that!